src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1406,21 +1406,75 @@ namespace cryptonote
     return true;
   }
   //-----------------------------------------------------------------------------------------------
+  bool core::notify_txpool_event(const epee::span<const cryptonote::blobdata> tx_blobs, const epee::span<const crypto::hash> tx_hashes, const epee::span<const cryptonote::transaction> txs, const std::vector<bool> &just_broadcasted) const
+  {
+    if (!m_zmq_pub)
+      return true;
+
+    if (tx_blobs.size() != tx_hashes.size() || tx_blobs.size() != txs.size() || tx_blobs.size() != just_broadcasted.size())
+      return false;
+
+    std::vector<txpool_event> results{};
+    results.resize(tx_blobs.size());
+
+    /* Publish txs via ZMQ that are "just broadcasted" by the daemon. This is
+       done here in addition to `handle_incoming_txs` in order to guarantee txs
+       are pub'd via ZMQ when we know the daemon has/will broadcast to other
+       nodes & *after* the tx is visible in the pool. This should get called
+       when the user submits a tx to a daemon in the "fluff" epoch relaying txs
+       via a public network. */
+    bool valid_events = false;
+    for (std::size_t i = 0; i < results.size(); ++i)
+    {
+      results[i].tx = std::move(txs[i]);
```

Comment:
You shouldn't be able to actually move these objects, its a `const` view of the transactions. I think this just results in the copy-constructor being called (which is annoying as the compiler doesn't actively complain).

Same thing on next line.

Reply:
And indeed, [this just results in a copy](https://stackoverflow.com/questions/28595117/why-can-we-use-stdmove-on-a-const-object). I have such a love/hate for C++.

Reply:
Ya this was odd of me to leave as is. Will fix.

Reply:
This never changed, but I don't think its a big deal (other than being slightly deceptive). It probably makes sense just to leave it, because if the `const`ness if the function ever changes this function call changes (although I think `transasction` does not have a proper `move` function).